api/api_test.go

```diff
@@ -103,11 +103,11 @@ func TestAPIQueryRangeGRPCCall(t *testing.T) {
 		t.Fatalf("Unexpected status code, expected 200, got %d", resp.StatusCode)
 	}
 
-	expectedContentType := "application/json"
-	gotContentType := resp.Header.Get("Content-Type")
-	if gotContentType != expectedContentType {
-		t.Fatalf("Unexpected Content-Type, expected %s, got %s", expectedContentType, gotContentType)
-	}
+	//expectedContentType := "application/json"
```

Comment:
Commented code?

Reply:
Will update this after I finish the tests